authentication/src/main/kotlin/com/uber/sdk2/auth/api/sso/SsoLink.kt

```diff
@@ -13,7 +13,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.uber.sdk2.auth.api.response
+package com.uber.sdk2.auth.api.sso
 
-/** Holds the access token that is returned after a successful authentication request. */
-data class AccessToken(val token: String, val scope: String, val expiresIn: Long)
+import kotlinx.coroutines.flow.Flow
+
+/**
+ * Represents the Single Sign-On (SSO) link for authentication. This class is used to start the SSO
+ * flow
+ */
+interface SsoLink {
+  suspend fun execute(optionalQueryParams: Map<String, String>): Flow<String>
```

Comment:
Missing KDoc

Reply:
There's something suspicious here. `execute` returns a `Flow`, but is also suspend. Tipically if you return `Flow`, you don't need `suspend`. Also, `execute` becomes a misnomer, because the actual code will only be executed when the flow is collected.